### DISCUSSION TITLE
what happened to the codespaces keynote

### DISCUSSION THREAD

**Discussion by mrdiamonddirt** [community/events#45] [Keynotes]
the codespaces keynote just ended abruptly and now the .net one did too?

**Comment by ossanna16** [comment]
Hi there @mrdiamonddirt! Sorry about that! The recordings will be available after the conference and you can also read a summary of the keynote announcements [here](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/).

**Reply by mrdiamonddirt** [reply]
Thanks for the reply, i thought i was going crazy ☺

**Reply by ossanna16** [reply]
Not at all 🙂
